src/components/test-result-header/test-result-header.html

```diff
@@ -0,0 +1,28 @@
+<ion-grid>
+    <ion-row>
+        <ion-col col-3 [ngClass]="summaryMetadata.selectedManoeuvre ? 'action-completed' : 'action-incomplete'">
+            <div [ngClass]="'test-section'">
+                <div [ngClass]="'test-section-text'">Manoeuvre</div>
```

Comment:
Perhaps it's better to do a standard `class` where the class name is fixed?

Reply:
Okay, I can do that. Keep it consistent